src/components/ToolMenu/Draw/Attributions/index.tsx

```diff
@@ -123,86 +185,129 @@ const AttributionDrawer: React.FC<AttributionDrawerProps> = ({
     }
   };
 
-  const getFormItems = () => {
-    const filteredProperties = currentProperties;
-
-    if (filteredProperties.geometry) {
-      delete filteredProperties.geometry;
-    }
+  const onValuesChange = (changedFields: FormData, fields: FormData) => {
+    updateAvailableFeatureAttributes(fields);
+  };
 
-    return Object.entries(filteredProperties).map(([key, value]) => {
+  const getFormItems = (fields: FormListFieldData[], rmFn: any) => {
+    return fields.map((field) => {
+      const removeAndMessage = () => {
+        notification.destroy();
+        notification.info({
+          message: (t('notificationDeleteText.title')),
+          description: (t('notificationDeleteText.info')),
+          placement: 'top',
+          duration: 6
+        });
+        remove(rmFn, field.name);
+      };
       return (
         <div
-          key={key}
+          key={field.key}
           className='attribute-row'
         >
           <AttributionRow
-            keyName={key}
-            key={key}
-            onChange={onKeyChange}
-          />
-          <MinusCircleOutlined
-            onClick={() => remove(key)}
+            keyName={field.name}
+            key={field.key}
+            onChange={onChange}
+            options={availableFeatureAttributes}
           />
+          <Popconfirm
+            title={t('AttributionPopConfirm.title')}
+            okText={t('AttributionPopConfirm.okText')}
+            cancelText={t('AttributionPopConfirm.cancelText')}
+            placement='topRight'
+            onConfirm={removeAndMessage}
+          >
+            {contextHolder}
+            <Button
+              className="remove-attribute-button"
+              // onClick={console.log('hi')}
+              type='primary'
+              danger={true}
+              icon={
+                <FontAwesomeIcon
+                  icon={faMinusSquare}
+                />
+              }
+            />
+          </Popconfirm>
         </div>
       );
     });
   };
 
   return (
-    <>
-      <Drawer
-        title={t('Attribution.title')}
-        className='attribution-drawer'
-        placement="right"
-        mask={false}
-        maskClosable={false}
-        onClose={handleClose}
-        {...passThroughProps}
-      >
+    <Drawer
+      title={t('Attribution.title')}
+      className='attribution-drawer'
```

Comment:
```suggestion
      className="attribution-drawer"
```

Reply:
done in: [1410fd6](https://github.com/terrestris/shogun-gis-client/pull/1506/commits/1410fd6bc66d3716fe9b90f51323beea1a3978ef)